plugin/src/ios/CDVQonversionPlugin.m

```diff
@@ -150,6 +150,38 @@ - (void)checkTrialIntroEligibilityForProductIds:(CDVInvokedUrlCommand *)command
     }];
 }
 
+- (void)remoteConfig:(CDVInvokedUrlCommand *)command {
+    __block __weak CDVQonversionPlugin *weakSelf = self;
+    [self.qonversionSandwich remoteConfig:^(NSDictionary<NSString *,id> * _Nullable result, SandwichError * _Nullable error) {
+        [weakSelf returnCordovaResult:result error:error command:command];
+    }];
+}
+
+- (void)attachUserToExperiment:(CDVInvokedUrlCommand *)command {
+    __block __weak CDVQonversionPlugin *weakSelf = self;
+    NSString *experimentId = [command argumentAtIndex:0];
+    NSString *groupId = [command argumentAtIndex:1];
+    [self.qonversionSandwich attachUserToExperimentWith:experimentId groupId:groupId completion:^(NSDictionary<NSString *,id> * _Nullable result, SandwichError * _Nullable error) {
+        if (error) {
+            [weakSelf returnCordovaResult:nil error:error command:command];
+        } else {
+            [weakSelf returnCordovaResult:@{} error:nil command:command];
+        }
```

Comment:
It won't send result if error exists anyway
```suggestion
        [weakSelf returnCordovaResult:@{} error:error command:command];
```

Reply:
It's not the same logic. `@{}` it's not the same as `nil`